components/collapsible/basic/src/index.scss

```diff
@@ -63,7 +72,8 @@
     }
 
     .sui-CollapsibleBasic-collapsibleContent {
-      max-height: 100%;
+      max-height: 100vh;
```

Comment:
Do you see a visible change when using vh? Just being curious.

Reply:
It does not change anything visible at all, but usign vh allows that the animations work when using the component in FC page, with `max-height: 100%` the transition does not wokk.

Reply:
👍